constants/ProjectThemes.ts

```diff
@@ -4,6 +4,16 @@ export interface IProjectTheme {
   name: string
 }
 
+export enum Themes {
+  VANGOUGH,
```

Comment:
Could use these `ENUMS` for themes, or 1,2,3,4 ...

Change the type for method when that happens, `string` is there for `test`ing

Reply:
yes makes sense